src/tile/Tile.js

```diff
@@ -107,7 +104,7 @@ const Tile = props => {
           styles.imageContainer,
           imageContainerStyle && imageContainerStyle,
         ]}
-        imageStyle={styles.imageStyle}
+        imageStyle={{ resizeMode: 'cover' }}
```

Comment:
This doesn't change the result haha 😅 The issue here is that `resizeMode` needs to be a props of the `BackgroundImage`, and not a style anymore
```jsx
<BackgroundImage
  …
  resizeMode="cover"
/>
```

Reply:
Oops - my bad. Silly mistake. 😅

Reply:
This modifications don't work to me.